content/en/blog/2022-02-09-security-the-value-of-sboms/index.md

```diff
@@ -0,0 +1,169 @@
+---
+
+## Flux - built with security in mind
+
+You don't get to re-architect a successful project very often, but we
+did about two years ago. The Flux project was already off to a great
+start and had [many happy adopters](/adopters/#flux-v1) and many of
+its design principles we kept at the forefront of our mind:
+
+- Pull vs Push: if you haven't read this [great blog
+  post](https://www.weave.works/blog/why-is-a-pull-vs-a-push-pipeline-important)
+  from 2018 about why you want Pull - all it says still holds true.
+- Least amount of privileges.
+- Reusing best practises, libraries and tools, e.g. client-go and helm
+  APIs.
+- And more we will explain further down the line.
+
+Why did we re-architect and rewrite Flux? Flux Legacy (v1) had been
+started Mid-2016 and while worked great and still does. It didn't quite
+benefit from more recent developments in the Kubernetes space like
+[controller-runtime](https://github.com/kubernetes-sigs/controller-runtime)
+because it pre-dated them significantly.
+
+Also rewriting Flux as a set of very targeted controllers as a unique
+opportunity to reduce the scope (and thus attack surface) of these
+individual sub-projects and make testing and debugging a lot easier.
+Re-usability as well.
+
+All of this said, we believe that a blog series about Flux and its
+[security](/docs/security/)
+considerations and features is in order and we will kick it off talking
+about SBOMs.
+
+### What is a SBOM?
+
+Since Flux release 0.26 we publish a SBOM for each of the individual
+controllers. We reported about this in the [accompanying monthly update
+blog post](/blog/2022/01/january-update/#-security-enhancements).
+
+So what is a SBOM? It's short for Software Bill of Materials. Wikipedia
+defines it as
+
+> A **software bill of materials** (SBOM) is a list of components in a
+> piece of software. Software vendors often create products by
+> assembling open source and commercial software components. The SBOM
+> describes the components in a product. It is analogous to a list of
+> ingredients on food packaging: where you might consult a label to
+> avoid foods that may cause allergies, SBOMs can help organizations or
+> persons avoid consumption of software that could harm them.
+>
+> The concept of a BOM is well-established in traditional manufacturing
+> as part of supply chain management. A manufacturer uses a BOM to track
+> the parts it uses to create a product. If defects are later found in a
+> specific part, the BOM makes it easy to locate affected products.
+
+For the Flux project we publish a Software Bill of Materials (SBOM) with
+each release. The SBOM is generated with
+[Syft](https://github.com/anchore/syft) in the
+[SPDX](https://spdx.dev/) format.
+
+![SPDX logo](featured-image.png)
+
+The `spdx.json` file is available for download on the GitHub release page
+e.g.:
+
+```shell
+curl -sL https://github.com/fluxcd/flux2/releases/download/v0.25.3/flux\_0.25.3\_sbom.spdx.json | jq
```

Comment:
Why the `\`? This works fine ` curl -sL https://github.com/fluxcd/flux2/releases/download/v0.25.3/flux_0.25.3_sbom.spdx.json | jq`